geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/iterators/DtgAgeOffTest.scala

```diff
@@ -0,0 +1,133 @@
+package org.locationtech.geomesa.accumulo.iterators
+
+import org.apache.accumulo.core.client.IteratorSetting
+import org.apache.accumulo.core.client.mock.MockInstance
+import org.apache.accumulo.core.client.security.tokens.PasswordToken
+import org.apache.accumulo.core.security.Authorizations
+import org.geotools.data.DataStoreFinder
+import org.geotools.factory.Hints
+import org.joda.time.{DateTime, DateTimeZone}
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.accumulo.TestWithDataStore
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore
+import org.locationtech.geomesa.accumulo.index.AccumuloFeatureIndex
+import org.locationtech.geomesa.features.ScalaSimpleFeature
+import org.locationtech.geomesa.security.SecurityUtils
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.locationtech.geomesa.utils.index.IndexMode
+import org.locationtech.geomesa.utils.text.WKTUtils
+import org.opengis.feature.simple.SimpleFeature
+import org.opengis.filter.Filter
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+
+@RunWith(classOf[JUnitRunner])
+class DtgAgeOffTest extends Specification with TestWithDataStore {
+
+  sequential
+
+  override val spec = "some_id:String,dtg:Date,geom:Point:srid=4326"
+
+  "DTGAgeOff" should {
+
+    def configAgeOff(ads: AccumuloDataStore, days: Int): Unit = {
+      val is = new IteratorSetting(5, "ageoff", classOf[KryoDtgAgeOffIterator].getCanonicalName)
```

Comment:
i don't think the vis iterators are in the config at all they just run no matter what. versioning is at 20.